### PR TITLE
Migrate precommit rules

### PR DESCRIPTION
**Pull request by davidbarna**
## Description
Proposal for sui-precommit new deal

Changes:
* **precommit rules are run a as a whole**
  * Pros:
    *  If you add new rules, we don't have to change installation or worry about chantes made by user
  * Cons:
     * User can't choose what to executed.
* **Now hook install is not automatic**
  * Pros:
     * User controls how he wants to integrate with git. He can even set the precommit before executing commitizen (no frustration)
     * No magic. User won't find its package.json to have been changed...
* **Husky is used for git hooks integration**
  * Pros: 
     * It's npm scripts based, with adds value to the project.
     * It's already used by sui-mono

Please review @miduga && @carlosvillu

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,14 +11,15 @@
...
-    "precommit": "npm run lint"
+    "precommit": "sui-precommit run"
   },
   "devDependencies": {
-    "@schibstedspain/sui-mono": "1",
     "@schibstedspain/sui-lint": "2",
-    "husky": "0.13.4",
+    "@schibstedspain/sui-mono": "1",
+    "@schibstedspain/sui-precommit": "2",
+    "husky": "^0.13.4",
```
Comment (miduga) [inline package.json]: Maybe better without caret.

File: packages/sui-precommit/bin/sui-precommit-install.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    installScript('lint', 'sui-lint js && sui-lint sass')
+    installScript('precommit', 'sui-precommit run')
+  })
+  .catch(function () {
+    log('sui-precommit could not be properly installed')
+    process.exit(1)
+  })
+
+/* eslint-disable no-console */
+function log (...args) { args[0] = '[sui-precommit] ' + args[0]; console.log(...args) }
```
Comment (miduga) [inline packages/sui-precommit/bin/sui-precommit-install.js]: Better in two lines without semicolon.
```
function log (...args) {
  args[0] = '[sui-precommit] ' + args[0]
  console.log(...args)
}
```

File: package.json (hunk 1 above)
Comment (davidbarna) [inline package.json]: Right ! Done.

File: packages/sui-precommit/bin/sui-precommit-install.js (hunk 2 above)
Comment (davidbarna) [inline packages/sui-precommit/bin/sui-precommit-install.js]: Done.

**Pull request merged**